src/passes/I64ToI32Lowering.cpp

```diff
@@ -372,14 +372,19 @@ struct I64ToI32Lowering : public WalkerPass<PostWalker<I64ToI32Lowering>> {
   }
 
   void visitGetLocal(GetLocal* curr) {
-    if (curr->type != i64) return;
-    curr->index = indexMap[curr->index];
+    if (curr->type != i64) {
```

Comment:
Overall lgtm, nice. But how about in the very first line of these two functions, we do
```
curr->index = indexMap[curr->index];
```
unconditionally? Easier to understand that way I think.